### ISSUE TITLE
Improve usability of chag

### ISSUE THREAD

**Issue by dpsenner** [dpsenner/chag#2]
The chag commands are currently not easy to remember because they mostly do not reflect what they actually do. This issue is about finding better names for the commands and further improve the descriptions of the commands.

**Comment by dpsenner** [comment]
The commands currently are:

```
Commands:
  contents Get the contents of a changelog entry.
  entries  List all versions in a changelog file.
  latest   Get the latest tag in a changelog.
  tag      Create an annotated git tag based on a changelog entry.
  update   Updates the version and date of the most recent changelog entry.
  create   Creates a new CHANGELOG.md file with a standard header.
  entry    Add a quick entry to the current version.
  next     Create a new unreleased version in changelog file.
```

I think that they should be renamed to the following while keeping the original ones as aliases. Below the wording and syntax of the descriptions are improved too.
```
Commands:
  init       Creates a new CHANGELOG.md file with the default header.
  add        Adds a new 'Unreleased' changelog entry.
  list       Gets a list of all changelog entries.
  latest     Gets the latest of the changelog entries.
  show       Shows the changes of a changelog entry. The command shows the latest changelog entry if not specified otherwise.
  addchange  Adds a change to the latest changelog entry.
  setversion Sets the version and date of the latest changelog entry.
  tag        Creates an annotated git tag based on a changelog entry.

Deprecated commands; these commands will be removed in a future major release:
  create     Creates a new CHANGELOG.md file with the default header. This command has been replaced by the init command.
  entries    List all versions in a changelog file. This command has been replaced by the list command.
  contents   Get the contents of a changelog entry. This command has been replaced by the show command.
  update     Updates the version and date of the most recent changelog entry. This command has been replaced by the setversion command.
  next       Create a new unreleased version in changelog file. This command has been replaced by the add command.
  entry      Add a quick entry to the current version. This command has been replaced by the addchange command.
```

**Comment by dpsenner** [comment]
Fixed with release [1.3.0](https://github.com/dpsenner/chag/releases/tag/1.3.0).
